run-tests.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+/app/bin/rake db:drop db:create spec:prepare
```

Comment:
This one I won't argue, I completely agree and I am already working on it, it was the fastest and hackiest way to have instructions on Testing, but this will at the very least go into a Rake Task, but I'm still trying to make docker-compose run everything by itself. Les commands, less complexity, more happiness.

The reason for the many lines is, beside sending the PR sooner, `spec/vcr`, which fails when we run `rspec`, so I simply copied what Travis runs to get the discussion going.

Ideally we shouldn't use patterns, if a test is supposed to fail we're supposed to take it off or move it to its feature branch.